admin-manual/installation-setup/upgrading/upgrading.rst

```diff
@@ -410,50 +410,81 @@ Update search indices
-Archivematica 1.12, you'll need to
-:ref:`recreate the indices <recreate-indices>`.
+The update can be accomplished one of two ways. Preferably, you can
+:ref:`reindex the documents <reindex-documents>` which is usually faster because
+the same documents that you already have indexed will be re-ingested. We would
+love to know if this is not working for you, but when that's the case, it is
+possible to :ref:`recreate the indices <recreate-indices>` which will take much
+longer to complete because it accesses the original data, e.g. your AIPs.
 
-.. _update-mappings:
+.. _reindex-documents:
 
-Update the Elasticsearch mappings
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Reindex the documents
+^^^^^^^^^^^^^^^^^^^^^
 
-Using this method, the index mappings will be modified in place. This will
-allow you to make the necessary updates quickly and without requiring a full
-reindex.
+In Elasticsearch, it is possible to add new fields to search indices but it is
+not possible to update existing ones. The recommended strategy is to create new
+indices with our desired mapping and reindex our documents. This is based on the
+`Reindex API`_.
 
-To perform the update, run the `update_elasticsearch_mappings` management task:
+.. warning::
+   Before you continue, we recommend to back up the Elasticsearch data. Please
+   read the official docs for instructions.
 
-Execution example:
+Assuming that your Elasticsearch cluster is available via ``127.0.0.1:9200``,
+this is how we can list existing indices:
 
 .. code:: bash
 
-   sudo -u archivematica bash -c " \
-       set -a -e -x
-       source /etc/default/archivematica-dashboard || \
-           source /etc/sysconfig/archivematica-dashboard \
-               || (echo 'Environment file not found'; exit 1)
-       cd /usr/share/archivematica/dashboard
-       /usr/share/archivematica/virtualenvs/archivematica-dashboard/bin/python \
-           manage.py update_elasticsearch_mappings
-   ";
+   $ curl -s -X GET 'http://localhost:9200/_cat/indices/%2A?v=&s=index:desc'
+   health status index         uuid                   pri rep docs.count docs.deleted store.size pri.store.size
+   yellow open   transfers     lYqkYjwZRy2XG8CP_3S3PQ   5   1          0            0      1.2kb          1.2kb
+   yellow open   transferfiles K5gnDZyOQz2JdIeZ6adJsQ   5   1          0            0      1.2kb          1.2kb
+   yellow open   aips          yAyK_koXThaZcWsBYfzN7w   5   1         17            0    101.4mb        101.4mb
+   yellow open   aipfiles      TVrrX8jkRhWWxGfvK_M6zg   5   1      11987            0      2.9gb          2.9gb
+
+Ensure that the Elasticsearch heap size is big enough to accomodate the size of
+the indices. The current size can be found under ``/etc/default/elasticsearch``
+(Ubuntu) or ``/etc/sysconfig/elasticsearch`` (CentOS):
+
+.. code:: bash
+
+   $ grep ES_JAVA_OPTS= /etc/default/elasticsearch
+   ES_JAVA_OPTS="-Xms2g -Xmx2g"
+
+For our example, it shoul be greater than 3G. Update ``ES_JAVA_OPTS`` as
```

Comment:
Typo: "shoul" should be "should"

Reply:
Typo aside, this is a really nice balance of lots of technical details while staying approachable. It really helps to have this example to work with.